tracer/src/Datadog.Trace/Generated/Datadog.Trace.SourceGenerators/Datadog.Trace.SourceGenerators.TagsListGenerator.TagListGenerator/TagAttribute.g.cs

```diff
@@ -0,0 +1,46 @@
+﻿// <auto-generated/>
+#nullable enable
+
+namespace Datadog.Trace.SourceGenerators;
+
+/// <summary>
+/// Used to designate a property as corresponding to the provided
+/// <see cref="TagName"/>. Should only be used in ITags classes.
+/// Used for source generation.
+/// </summary>
+[System.AttributeUsage(System.AttributeTargets.Property, AllowMultiple = false)]
+internal sealed class TagAttribute : System.Attribute
```

Comment:
I'm curious, is there a reason that this was auto-generated instead of manually written?

Reply:
_Generally_, no, but it does give one nice feature - if the source generator isn't working at all for some reason, then this attribute won't be part of the compilation, and it will be obviously flagged by the compiler. In this case, our code wouldn't compile either way, but that wouldn't have to be the case, plus the errors might be a bit more obtuse.